### ISSUE TITLE
Suggestion: Use Caddy built-in file browser as an alternative

### ISSUE THREAD

**Issue by maple3142** [wahyd4/aria2-ariang-docker#242]
Currently, caddy's builtin file browser can be enabled by changing `Caddyfile` to:

```caddyfile
@protected path / /index.html /js/* /files/*

basicauth @protected {
        ...
}

# omitted...

route /files/* {
    uri strip_prefix /files
    file_server {
        root /data
        browse
    }
}
```

and to get `Content-Type` works correct (so that images and videos can be viewed or played in browser), mime database [required by go](https://pkg.go.dev/mime#TypeByExtension) needs to be avaiable (e.g. mounting `/usr/share/mime/globs2`)

Of course this is missing some feature that the current file browser supports (e.g. sharing), but it is good for people who want direct HTTP based file access.

The UI looks like this

![image](https://github.com/wahyd4/aria2-ariang-docker/assets/9370547/2ac3770d-d73a-4ea4-8ca3-aedfc87ca596)

**Comment by wahyd4** [comment]
Cool, it sounds good, then we can make filebrowser optional. 
Would your like to make a PR for this?

**Comment by maple3142** [comment]
Of course

I am wondering should this be in a different path than `/files` (maybe `/fs`) that it can coexist with existing file browser, and both filebrowser and caddy file_server can be enabled/disabled separately? I am not sure how to handle this with current multiple Caddyfile setup.

**Comment by wahyd4** [comment]
Yeah, I think we can use `/fs` for the Caddy file server one. 
Regarding the multi Caddyfile setup, most of them are the same, the only difference is `SecureCaddyFile` will be used when user turns on authentication by enabling `ENABLE_AUTH`.

**Comment by maple3142** [comment]
Since `/ro` (caddy's file listing) does not come with authentication by defaut, it needs to be protected in `HerokuCaddyfile`. `/js/` paths should also be protected because it contains the aria2 secret.

```caddy
@protected path / /index.html /js/* /ro/*

basicauth / {
	ARIA2_USER ARIA2_PWD_ENCRYPT
}
```

**Comment by wahyd4** [comment]
Yeah, that makes sense, but does `@protected path / /index.html /js/* /ro/*` actually work? Can you share the doc?
